scripts/download_checkpoint.py

```diff
@@ -1,21 +1,13 @@
 """Download checkpoint."""
 import argparse
 import os
+import subprocess
```

Comment:
Sorry, this file is deprecated. Could you delete the whole file?

Reply:
Done!